packages/kbn-i18n/GUIDELINE.md

```diff
@@ -84,18 +84,24 @@ In case when `indicesLength` has value 1, the result string will be "`1 index`".
 
 #### In ReactJS
 
+The long term plan is to rely on using `FormattedMessage` and `i18n.translate()` by statically importing `i18n` from `@kbn/i18n` package. **Avoid using `injectI18n` and rely on `i18n.translate()` instead.**
+
 - You should use `<FormattedMessage>` most of the time.
-- In case when the string is expected (`aria-label`, `placeholder`), use `props.intl.formatmessage()` (where `intl` is  passed to `props` by `injectI18n` HOC).
-- In case if none of the above can not be applied (e.g. it's needed to translate any code that doesn't have access to the component props), you can call JS function `i18n.translate()` from `@kbn/i18n` package.
+- In case when the string is expected (`aria-label`, `placeholder`), Call JS function `i18n.translate()` from `@kbn/i18n` package.
+
+Currently we support the following ReactJS `i18n` tools, but will be removed in future releases:
+- Usage of `props.intl.formatmessage()` (where `intl` is  passed to `props` by `injectI18n` HOC).
 
 #### In AngularJS
 
-- Use `i18n` service in controllers, directives, services by injected it.
+The long term plan is to rely on using `i18n.translate()` by statically importing `i18n` from `@kbn/i18n` package. **Avoid using the `i18n` filter and the `i18n` service injected in controllers, directives, services.**
+
+- Call JS function `i18n.translate()` from `@kbn/i18n` package.
 - Use `i18nId` directive in template.
-- Use `i18n` filter in template for attribute translation.
-- In case if none of the above can not be applied, you can call JS function `i18n.translate()` from `@kbn/i18n` package.
 
-Note: Use one-time binding ("{{:: ... }}") in filters wherever it's possible to prevent unnecessary expression re-evaluation.
+Currently we support the following AngluarJS `i18n` tools, but will be removed in future releases:
+- Usage of `i18n` service in controllers, directives, services by injected it.
```

Comment:
Should this be "injected by it" instead of "by injected it'?

Reply:
```suggestion
- Usage of `i18n` service in controllers, directives, services by injecting it.
```